chain/chunks/src/lib.rs

```diff
@@ -909,6 +906,39 @@ impl ShardsManager {
         Ok(header)
     }
 
+    pub fn insert_chunk_forward(&mut self, forward: PartialEncodedChunkForwardMsg) {
```

Comment:
would `insert_forwarded_chunk` be a better name?

Reply:
Done in https://github.com/nearprotocol/nearcore/pull/2769/commits/1e284e0b3d0ac42a5af9d817ee6b9b10210ffff6